.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
+  "trailingComma": "es5"
```

Comment:
Not sure on the general convention of things but personally I would always opt for `arrowParens: "always"` as well, but of course it's personal preference. Just a thought!